app/src/main/java/com/hedvig/app/util/FeatureFlag.kt

```diff
@@ -0,0 +1,17 @@
+package com.hedvig.app.util
+
+import com.hedvig.app.BuildConfig
+
+enum class FeatureFlag(
+    val key: String,
+    val title: String,
+    val explanation: String,
+    val enabled: Boolean = false,
+) {
+    MOVING_FLOW(
+        "moving_flow",
+        "Moving Flow",
+        "Lets a user change their address and get a new offer",
+        BuildConfig.BUILD_TYPE == "debug"
```

Comment:
You can also use `isDebug()` from `ApplicationModule.kt`, which would enable this button in both dev- and staging-builds 🙂